api/src/main/java/gov/cms/ab2d/api/service/JobServiceImpl.java

```diff
@@ -37,6 +37,16 @@ public Job createJob(String resourceTypes, String url) {
         return jobRepository.save(job);
     }
 
+    public void cancelJob(String jobID) {
+        Job job = getJobByJobID(jobID);
+
+        if (job.getStatus() == JobStatus.CANCELLED || job.getStatus() == JobStatus.SUCCESSFUL || job.getStatus() == JobStatus.FAILED) {
```

Comment:
We can make this cleaner by adding `isCancellable` attribute to the `JobStatus` enum and checking that instead.